ironfish/src/testUtilities/matchers/blockchain.ts

```diff
@@ -50,10 +50,27 @@ async function toAddBlock(self: Blockchain, other: Block): Promise<jest.CustomMa
   return makeResult(true, `Expected to not add block at ${String(other.header.sequence)}`)
 }
 
+async function toAddDoubleSpendBlock(
+  self: Blockchain,
+  other: Block,
+): Promise<jest.CustomMatcherResult> {
+  // Mock nullifiers to allow creation of a double spend chain
+  const containsMock = jest.spyOn(self.nullifiers, 'contains').mockResolvedValue(false)
+  const result = await self.addBlock(other)
+  containsMock.mockRestore()
+
+  if (!result.isAdded) {
+    return makeResult(false, `Could not add block: ${String(result.reason)}`)
+  }
+
+  return makeResult(true, `Expected to not add block at ${String(other.header.sequence)}`)
```

Comment:
This is doing some very scary things that shouldn't be allowed by our consensus layer and corrupting the test chain DB. What is this used for? We should fix any test so this never gets added here.